index.css

```diff
@@ -0,0 +1,161 @@
+html,
+body {
+    margin: 0;
+    font-family: 'Karla', sans-serif;
+    background-color: #219ebc;
+}
+
+/*typography*/
+
+h1,
+h2,
+h3 {
+    text-align: center;
+}
+
+h1,
+h2 {
+    color: #EED6D3;
+    margin: 0;
+}
+
+h1 {
+    font-size: 28px;
+}
+
+h2 {
+    font-size: 16px;
+}
+
+h3 {
+    margin: 25px;
+}
+
+/* layout */
+
+header {
+    background-color: #67595E;
+}
+
+.header-inner {
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+    width: 370px;
+    gap: 15px;
+    padding: 10px 0 0 0;
+}
+
+.pumpkin-img {
+    width: 80px;
+}
+
+.controls-container {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    margin: 0px auto;
+}
+
+/* controls */
+
+.emotion-radios {
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #bcbcbc;
+    border-radius: 5px;
+}
+
+.radio {
+    display: flex;
+    align-items: center;
+    border-bottom: 1px solid #bcbcbc;
+    padding: 4px 8px;
+}
+
+label {
```

Comment:
you already have a class for this `gifs-check-label`, I'd put these styles in that rather then styling all labels

Reply:
In my js file in the renderEmotionRadios function, I am rendering out a template literal for each of the emotions in the cats array. Each of these emotions has a radio input and label that is generated. So I need this general label callout to style each of the radio inputs and their labels. This gifs-check-label class is for the "animated gifs only" label only.